scripts/cast-on-tenderly/index.js

```diff
@@ -87,7 +87,7 @@ const runSpell = async function () {
     const eta = await spell.eta();
 
     console.info(`warping the time to "${eta}"...`);
-    const currentUnixTimestamp = Date.now() / 1000;
+    const currentUnixTimestamp = Math.floor(Date.now() / 1000);
```

Comment:
Alright, let's see if it always works. I previously had troubles with `evm_increaseTime` and [`premature-exec`](https://github.com/dapphub/ds-pause/blob/0763eafcf926fd2e073aee5f047f3decb842231c/src/pause.sol#L98), but maybe it was because of this exact problem that you're now fixing